### PR TITLE
Add Sourcemap header

### PR DESCRIPTION
**Pull request by Elchi3**
Here I'm not sure if we want "SourceMap" or "Source maps". I think more people talk about it as "Source maps"

Edit: As Patrick notes, this is just the header and not general source maps support

### CODE REVIEW

File: features/sourcemaps.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+name: Source Maps
+description: The `SourceMap` HTTP response header links generated code to a source map, enabling the browser to reconstruct the original source and present the reconstructed original in the debugger.
```
[the remark this answers is not in the document]
Comment (Elchi3) [reply features/sourcemaps.yml:2]: > Does support for the header mean that the browser actually uses the source map in the debugger?

In BCD it is sometimes weird to talk about compatibility of HTTP features. In this case, if the browser debugger doesn't actually do anything with source maps, I would encourage an update to BCD.

> The spec says sourcemap: … and MDN has SourceMap: …. Do we care about case? Does it matter?

HTTP header names are not case sensitive.

File: features/sourcemaps.yml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+name: Source maps
```
Comment (captainbrosset) [inline features/sourcemaps.yml:1]: I think we should make this name more specific to the fact that this feature is about the HTTP header. There are multiple ways to load source maps into DevTools, and using the HTTP response header is one of them.

Other ways include:

* By including comments in JS: `//# sourceMappingURL=...`
* Automatically, by using [Edge DevTools' Symbol Server integration](https://learn.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/javascript/publish-source-maps-to-azure).
* Manually in DevTools.

Maybe change the name to "Source map header"?

Comment (Elchi3) [reply features/sourcemaps.yml]: Thanks Patrick, in this case, yes, I think the correct thing would be to make this feature just about the header.

**Review by ddbeck — APPROVED** [review]
I'm happy to see this merge, but I'll leave it to Patrick or Florian to decide whether to merge with or without Patrick's re-review. 😄

**Pull request merged**